packages/core/src/common/utils/find-key-contains.injectable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+
+export type FindKeyContains = <T>(map: Map<string, T>, text: string) => T | undefined;
+
+const findKeyContainsInjectable = getInjectable({
+  id: "find-key-contains",
+
+  instantiate: (): FindKeyContains => {
+    return <T>(map: Map<string, T>, text: string) => {
+      const entries = map.entries();
+
+      for (const [key, value] of entries) {
+        if (key.includes(text)) {
```

Comment:
instead of using `includes` we should try to reconstruct the whole legacy key path.

something like this:
`path.join(directoryForUserData, name, "package.json") `

Reply:
done